db/schema.rb

```diff
@@ -301,6 +302,7 @@
     t.boolean "terms_and_conditions_agreed"
     t.boolean "data_processing_agreed"
     t.boolean "consent_response_confirmed"
+    t.boolean "observed_session_agreed"
```

Comment:
I wonder if this might read better if it were `agreed_to_user_research_observation`

Reply:
`user_research_observation_agreed` matches the other fields which all end in `_agreed`, shall I do that instead?